input/pagecontent/use-cases.md

```diff
@@ -1,7 +1,18 @@
-## AU eRequesting R1 use cases
-This page contains the use cases for AU eRequesting for the first release of the TDG Implementation Guide (R1). This is a working page and will reflect the current state of agreement by the TDG as well as the current draft of the use case material.
+In the context of AU eRequesting, use cases focus on standirdising clinical requesting and ordering within the Australian healthcare system, specifically targeting Pathology and Imaging requests in AU eRequesting Release 1 (R1). The use cases aim to ensure that the system supports accurate and efficient data exchange and workflow management.
```

Comment:
1. Replace "In the context of AU eRequesting, use cases" with "The use cases for AU eRequesting"
1. Replace "Release 1 (R1)" with "R1".
2. Replace "The use cases aim to ensure the system supports" with "These use cases aim to ensure systems support"

Reply:
Done